README.md

```diff
@@ -37,20 +37,14 @@ If you want to start a REPL with the correct environment already activated, use
 julia --project=. -t 4
 ```
 
-### Global dependencies
-CarboKitten has some dependencies that are only needed for developing, but not for using the library. You may want to install those in the global environment:
-
-```shell
-julia -e 'using Pkg; Pkg.add.(["DaemonMode", "LiveServer", "Revise"])'
-```
-
 ### Examples
 
-You'll get the best experience by running examples from the Julia REPL. There are however also some example scripts, that should work stand-alone.
+You'll get the best experience by running examples from the Julia REPL. There are however also some example scripts that should work stand-alone.
 
 ```shell
-julia --project=. examples/bosscher-schlager-1992.jl
+julia --project=. examples/ca-with-prod.jl
 ```
+This command will write the output in the HDF5 format into the `data` folder.
```

Comment:
Perhaps add e.g.: "You can check that output is written there after executing this command.".